codecov.yml

```diff
@@ -1,6 +1,16 @@
+codecov:
+  require_ci_to_pass: true
+
 coverage:
   precision: 2
-  round: nearest
-  range: "40...70"
+  round: down
+  range: "70...100"
   ignore:
-    - "com.garret.movies.moviestore.MovieStoreApplication.java"
\ No newline at end of file
+    - "dao/.*"
+    - "movie-service/src/main/java/com/garret/movies/service/api/.*"
```

Comment:
@Garret-vin @Dominys Instead of exclusion here configure exclusion for jacoco coverage report